packages/dev/docs/pages/releases/2022-05-03.mdx

```diff
@@ -0,0 +1,224 @@
+
+---
+description: DatePicker and Calendar are now in RC, complete with new documentation! Other updates include bug and accessibility fixes for our existing components. Thank you to all who contributed to this release!
+date: 2022-05-03
+---
+
+# May 3, 2022 Release
+We are pleased to announce that the `datepicker` and `calendar` packages have been release into rc! This update comes complete with new documentation for the [DatePicker](https://react-spectrum.adobe.com/react-spectrum/DatePicker.html), [DateRangePicker](https://react-spectrum.adobe.com/react-spectrum/DateRangePicker.html), [DateField](https://react-spectrum.adobe.com/react-spectrum/DateField.html), [TimeField](https://react-spectrum.adobe.com/react-spectrum/TimeField.html), [Calendar](https://react-spectrum.adobe.com/react-spectrum/Calendar.html) and [RangeCalendar](https://react-spectrum.adobe.com/react-spectrum/RangeCalendar.html) components and associated [React-Aria](https://react-spectrum.adobe.com/react-aria/useDatePicker.html) hooks.
+
+Other updates include bug and accessibility fixes for our existing components. Thank you to all who contributed to this release!
+
+
+## Fixes
+- Fix `useNumberField` to propagate name/placeholder props - [@MeowIng44](https://github.com/MeowIng44) - [PR](https://github.com/adobe/react-spectrum/pull/2884)
+- `useSelect` should call `onFocusChange` when focus is changed - [@lishichengyan](https://github.com/lishichengyan) - [PR](https://github.com/adobe/react-spectrum/pull/2970)
+- Adjust `Dialog` to better position footers, buttons, and headers - [@ktabors](https://github.com/ktabors) - [PR](https://github.com/adobe/react-spectrum/pull/2820)
+- Prevent `focusedKey` from scrolling into view on click/press - [@LFDanLu](https://github.com/LFDanLu) - [PR](https://github.com/adobe/react-spectrum/pull/2725)
+- Deprecate `placeholder` prop on most form components - [@reidbarber](https://github.com/reidbarber) - [PR](https://github.com/adobe/react-spectrum/pull/2966)
+- Manage focus when content changes using `Breadcrumbs`  - [@majornista](https://github.com/majornista) - [PR](https://github.com/adobe/react-spectrum/pull/2749)
+- Add try-catch to `isRTL` and fallback to en-US - [@BartoszGamza](https://github.com/BartoszGamza) - [PR](https://github.com/adobe/react-spectrum/pull/3030)
+- Fix ts missing generic arg with `SelectAria`  in `useSelect` - [@fitiskin](https://github.com/fitiskin) - [PR](https://github.com/adobe/react-spectrum/pull/3069)
+
+## Documentation
+- Add Spectrum docs link for `ContextualHelp` - [@matthewdeutsch](https://github.com/matthewdeutsch) - [PR](https://github.com/adobe/react-spectrum/pull/2979)
+- Add `DatePicker` and `Calendar` docs - [@devongovett](https://github.com/devongovett) - [PR](https://github.com/adobe/react-spectrum/pull/2873)
+
+## Under construction
+Pre-release versions of the following components have been released. Please feel free to try them out, and report any issues you encounter.
+
```

Comment:
Do we wanna call out ListView updates/new alpha version or are we saving that for a ListView centric release?

Reply:
Same question for other alpha/beta bumps (TagGroup, ColorArea)

Reply:
Still debating this question because I wanted to call out Datepicker specifically here

Reply:
None of those have docs so seems kinda pointless

Reply:
And since we have no links for docs just yet it didn't seem that helpful

Reply:
Fair enough